client/my-sites/invites/controller.js

```diff
@@ -59,6 +59,7 @@ export function acceptInvite( context, next ) {
 		<>
 			<AcceptInviteTitle />
 			<InviteAccept
+				userInvitePasswordless={ true }
```

Comment:
Keeping it here for explicitly only.

Reply:
I don't think we need this, switching to passwordless is really straightforward, let's just pass `isPasswordless` true in these 2 components: https://github.com/Automattic/wp-calypso/pull/87083/files#diff-50553278096846b3ca70c3e0a54f4bf683276b95a5523edb1c0e43c445db960cR26 and https://github.com/Automattic/wp-calypso/pull/87083/files#diff-49cf27d5bf48a0b877a5e5df736a831e80361f4f12b7172427d39cbddda468c3R181

Reply:
Removed